gr-blocks/grc/blocks_message_debug.block.yml

```diff
@@ -2,6 +2,15 @@ id: blocks_message_debug
 label: Message Debug
 flags: [ python, cpp ]
 
+parameters:
+-   id: en_uvec
+    label: Print PDU Vectors?
```

Comment:
probably don't need the question mark in the label - i don't see it in any other gr blocks bool parameters

Reply:
good call. it's been changed.